### DISCUSSION TITLE
Strelka save only one copy of file

### DISCUSSION THREAD

**Discussion by palichx** [Security-Onion-Solutions/securityonion#3445] [Ideas] [answered]
Hi,

again about strelka. In my Org we have some files wich starting from network share. in this case in each time copy of file will saved on disk. in result i have a lot of copys of the same file:
![image](https://user-images.githubusercontent.com/80513304/111187280-ea8b1600-85c4-11eb-8201-42d3e2e6e1c2.png)

![image](https://user-images.githubusercontent.com/80513304/111187457-0d1d2f00-85c5-11eb-8686-2d357e2718f4.png)

i think, will be nice save only one copy of identical files. 

thanks!

**Comment by weslambert** [comment, marked as answer]
Thanks for the feedback!  This is something we've wanted to look into in the past, but haven't had a chance to dedicate cycles to it. We are tracking this via https://github.com/Security-Onion-Solutions/securityonion/issues/3458.

**Reply by palichx** [reply]
thanks!
